### DISCUSSION TITLE
Star Trek Armada 1 Win10, post GOG release

### DISCUSSION THREAD

**Discussion by ScrewYouGitHub** [elishacloud/dxwrapper#170] [General]
Sorry for my username, I created it because it was at a time when GitHub made me mad for having to create this account, I think it was for either a Fallout New Vegas mod or something else that idgaf about anymore.

Since 2020 I have off and on replayed STA1 with elishacloud's Armada fix for Win10 along with the 1.2 and 1.3 patches on two different computers (PC and laptop) both using Win10, within the span of two years, until late 2022 when I had to remove Armada from my PC to get as much space as possible for large games. Recently after getting a 1TB M.2 NVMe, I have been trying to play Local/Multiplayer with my nephew, in short, my PC can no longer play Armada 1 like it used it with or without the fixes, however with the laptop that I used before my PC, this laptop can play and run STA1 with no issues, even after installing Millenium Project 1.9 the game still runs and plays.

I also downloaded elishacloud's "dxwrapper" zip, the one that is not "Armada-fix", and I could not understand it all, I followed the instructions and tested some of the dll's that are in the file folder, one dll (d3d9, I think) ran the game but it was like it was 320p or 480p or it was poorly ran through an AI upscaler. With this dll it worked until I tried to play STA1 with Millenium Project 1.9, again all of this is happening on my PC.

I was forced to buy the GOG version since the Disc version and the abandonware versions no longer function on my PC either, the GOG version works as it should but in exchange I do not have the ability to play multiplayer with the disc version while using GameRanger, not relevant to the STA1 fixes but the GOG version is missing the WON (World Opponent Network) capability, I copied and pasted those features from the disc version onto the GOG version and no change, I guess there's more to it than just copy and paste.

I will end with this.
Is there anything that I can do to make the original version or even the abandonware versions to work on my PC?

**Comment by mirh** [comment]
Try with these
https://github.com/elishacloud/dxwrapper/issues/121#issuecomment-820175971
https://github.com/elishacloud/dxwrapper/issues/49#issuecomment-1223423601

**Reply by ScrewYouGitHub** [reply]
I need to stop reading all of these when it's late and I'm tired, because so far, none of it makes sense to me when reading.

In my post I mentioned that I messed around with the settings.ini via elishacloud's "dxwrapper" ZIP and only one setting worked, which was the d3d9.

That thread that leads to the Mirror's Edge post is similar to what I've experienced with Armada disc, so I may have to try that solution, when I can.

**Reply by elishacloud** [reply]
I just posted an update to the Star Trek 1 fix.  Check it out here:  https://github.com/elishacloud/dxwrapper/wiki/Star-Trek-Armada-1
